Documentation/Signoffs/Extra.md

```diff
@@ -0,0 +1,96 @@
+# Extra
+
+![System](../Images/Extra_jrneal20.png)
+
+Figure 1: Extra subsystem and the pause switch can be seen in this figure.
+
+- The goal of this subsystem is to adhere to the rules as provided by the customer, Devcom, and to add lights and sounds to the design which will add to the point total as provided by Devcom. The system needs a pause switch to deactivate it between rounds. The system will also include lights and sounds before launching a projectile to add to the decoration portion of the design.
+
+## **Constraints:**
+
+| **Number:** | **Constraint:** | **Origin:** | 
+| --- | --- | --- |
+| 1. |  The interceptor shall have a switch that sets the system into a pause state that will keep the interceptor from firing. | Rulebook |
+| 2. | The voltage switched by the pause switch shall be 5V. | System Constraint|
+| 3. | The interceptor must have lights that are bright enough to be seen by the judges approximately six and a half feet away, which is the longest length of the gameboard. These lights must also be powered by a 5V source from the processor block and must emit a light that falls within the visual light spectrum of 380 to 720 nanometers [1]. | Rulebook / System |
+| 4. | The interceptor must make sounds before firing. The sounds will be powered by a voltage of 5V and the sounds will need to fall within the range of frequency humans can hear which is 20 to 20,000Hz [2]. The volume, or loudness, must fall within a range of 0, the threshold, to a maximum of 120dB and must not exceed 85dB for more than eight hours or it may cause hearing damage [2]. The buzzer should be louder than average room noise which is 30-50 dB [2]. | Rulebook / Safety |
```

Comment:
What does 0, the threshold, mean? 

This constraint still doesn't actually constrain the system along the important lines and instead tries to require that the buzzer operate on 5 volts. That's not a legitimate constraint. Remove it. In its place, give the constraints for this system. Below, provide the clear reasoning behind the constraint.